schematic_test.go

```diff
@@ -144,8 +148,27 @@ func TestMultipleEventBatches(t *testing.T) {
 }
 
 func TestCheckFlagOfflineMode(t *testing.T) {
-	client := schematic.NewClient("", schematic.WithDefaultFlagValues(map[string]bool{"test-flag": true}))
+	client, _ := mocks.NewClientWithMockHTTP("", gomock.NewController(t))
```

Comment:
Using mock http here so that the test fails if any requests are attempted during offline mode.

Reply:
It looks like there is a problem with tests failing

Reply:
yep sorry